dist/index.js

```diff
@@ -55283,7 +55302,9 @@ class Config {
     }
 
     if (this.input.mode === 'start') {
-      if (!this.input.ec2ImageId || !this.input.ec2InstanceType || !this.input.subnetId || !this.input.securityGroupId) {
+      const isSet = param => param;
+      const params = [this.input.ec2ImageId, this.input.ec2InstanceType, this.input.subnetId, this.input.securityGroupId];
```

Comment:
I was a bit confused about where this line came from: it's from #2, and appears here because the dist got rebuilt.